util/patch/patch_test.go

```diff
@@ -138,6 +140,36 @@ func TestHelperPatch(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "Only add ownerref update to unstructured object",
```

Comment:
This test passes even without your code change. Is that expected?

Reply:
I added a new one that actually tests this behavior

Reply:
(Both should be valid tests though)